code/Data/Drasil/Units/Chemistry.hs

```diff
@@ -1,7 +1,6 @@
 module Data.Drasil.Units.Chemistry where
 
-import Language.Drasil
-import Data.Drasil.SI_Units
+import Data.Drasil.SI_Units (mole, kilogram)
 
```

Comment:
This file's contents are not used anywhere at the moment, which is why removing import of "Language.Drasil" didn't break anything. (That, and multiple definitions of `molarityU` on lines 8 and 9).

Can this file (i.e. Data.Drasil.Units.Chemistry) be removed? Or should it be treated as though it is being used somewhere (i.e. reinsert import, rename `molarityU` variables)  @JacquesCarette @szymczdm

Reply:
This file should not be removed as it's relevant information (for a different domain -- we had an example that was chemistry related, but adding it has taken a backseat for the time being).

The multiple declarations of molarity are a problem. I'm not sure why they're both here with the same name and uid.

Reply:
I can update that and this PR will be ready for merge.